charts/keycloakx/templates/statefulset.yaml

```diff
@@ -71,7 +71,7 @@ spec:
         - name: keycloak
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}{{- if (.Values.image.digest) -}} @{{ .Values.image.digest }}{{- else -}}:{{ .Values.image.tag | default .Chart.AppVersion }} {{- end }}"
```

Comment:
Is the empty-space between the image and the `@` on purpose? I'd say that should not be there

Reply:
The whitespace shouldn't matter because of the '-}}' before it. That trims whitespace on that side. Nevertheless I just removed the unnecessary spaces.